packages/perseus-editor/src/components/__tests__/interactive-graph-settings.test.tsx

```diff
@@ -61,8 +61,10 @@ describe("InteractiveGraphSettings", () => {
         );
 
         // Act
-        const select = screen.getByRole("combobox", {name: "Ruler label:"});
-        await await userEvent.selectOptions(select, ["cm"]);
+        const select = screen.getByRole("button", {name: "Ruler label:"});
+        select.click();
```

Comment:
I removed the `userEvent.selectOptions` because WB dropdown isn't a combobox. 

I used `select.click()` instead of `userEvent.click()` because I was under the incorrect assumption that it was more straightforward to use click. But your question prompted me to read the userEvent docs, and now I think it makes more sense to change it. I'll bring back userEvent!